### PR TITLE
Second tutorial about time dependent data

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

Add the second tutorial about time-dependent data

## Todo
- [x] Link first tutorial
- [x] Add to "Getting started"
- [x] xarray examples -> mean, min, max

### CODE REVIEW

**Comment by vhirtham** [comment]
**Notes**
- plot first
- small data processing example (mean value)

**Comment by CagtayFabry** [comment]
can you update the `single_pass_weld` schema in a separate PR to point to `TimeSeries` @vhirtham ?

Or maybe we should allow both for now?

**Comment by vhirtham** [comment]
> can you update the `single_pass_weld` schema in a separate PR to point to `TimeSeries` @vhirtham ?
> 
> Or maybe we should allow both for now?

I will create a separate PR pointing to the `master` branch

**Comment by vhirtham** [comment]
@marscher @CagtayFabry no clue why deepsource is complaining here. The function is in the file and I didn't touch it either. Any clue? If not I will ignore it for now. If it keeps showing up, I will add a flag for deepsource to ignore it, before we merge everything into master.

File: tutorials/01_02_time_dependent_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %matplotlib widget"
+   ]
```
[the remark this answers is not in the document]
Comment (vhirtham) [reply tutorials/01_02_time_dependent_data.ipynb:118]: Didn't work. Solution postponed

**Pull request merged**